.github/workflows/welcome.yml

```
@@ -0,0 +1,33 @@
+    types: [opened]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/first-interaction@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.12)

[error] input "repo-token" is not defined in action "actions/first-interaction@v3". available inputs are "issue_message", "pr_message", "repo_token"

(action)